use std::error::Error;
pub fn show_help(_dirs: Vec<&str>) -> Result<String, Box<dyn Error>> {
    todo!()
}